src/lib/js/HeaderComponentModule.js

```diff
@@ -0,0 +1,10 @@
+function headerAnimattionHandler() {
```

Comment:
Need to look into this issue, The JavaScript method is not being called by the NavBar Component.
May you can create an issue regarding this problem.